lib/move-sync/index.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+// most of this code was written by Andrew Kelley
+// licensed under the BSD license: see
+// https://github.com/andrewrk/node-mv/blob/master/package.json
+// This is the sync version that somehow follows the same pattern.
```

Comment:
I'd say just remove this comment; by now, this is going to barely resemble the original code.